functions/src/firestore-types/lobby.ts

```diff
@@ -36,12 +36,46 @@ export type Player = {
   promptAnswer?: string;
 };
 
+export type LobbySettings = {
+  /**
+   * the number of catfish that are in play
+   */
+  catfishAmount: number;
```

Comment:
this comment makes it sound like if a catfist dies then this number will go down